lab_6_pipeline/pipeline.py

```diff
@@ -96,6 +167,8 @@ def get_cleaned_sentence(self) -> str:
         """
         Returns the lowercase representation of the sentence
         """
+        conllu_token = ConlluToken(self._text)
```

Comment:
you should have a list of tokens in `self._tokens` and construct final cleaned version of sentence based on them

Reply:
done